docs/source/metrics.rst

```diff
@@ -0,0 +1,141 @@
+=======
+Metrics
+=======
+BentoML now supports all metrics found in Prometheus Client. 
+User can create custom metrics and attach them to API endpoints and access them through the ``/metrics`` endpoint.
+
+
+Creating Metrics
+----------------
+First thing to note about creating a metric is how to access them. 
+You can access metrics directly from the ``bentoml`` package. 
+
+Counter
+~~~~~~~~~~~~~~~
+.. code-block:: python
+
+    from bentoml import metrics
+    # from bentoml.metrics import Counter -- you can also impprt Histogram directly
+    counter = metrics.Counter(...)
+
+
+Histogram
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
+    histogram = metrics.Histogram(...)
+
+
+Summary
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
+    summary = metrics.Summary(...)
+
+
+Gauge
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
+    gauge = metrics.Gauge(...)
+
+
+Info
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
+    info = metrics.Info(...)
+
+
+Enum
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
+    enum = metrics.Enum(...)
+
+
+Metric
+~~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    from bentoml import metrics
```

Comment:
I don't think this section is useful, maybe you can create a api_reference docs for `bentoml.metrics`.

API docs lives under `docs/source/reference`.

Reply:
Partially completed ✅